src/Nethermind/Nethermind.Evm/Tracing/ITxTracer.cs

```diff
@@ -127,7 +125,17 @@ public interface ITxTracer : IWorldStateTracer
         /// </remarks>
         bool IsTracingFees { get; }
 
-        bool IsTracing { get; }
+        bool IsTracing => IsTracingReceipt
```

Comment:
Also implement on base type in a cached way?
```
bool IsTracing {get} = ...
```
Otherwise is 11 interface calls per call

Reply:
"Cannot access non-static property 'IsTracingReceipt' in static contex"